build.py

```diff
@@ -307,8 +322,12 @@ def do_make(target=None, log=False):
                 kimages.append(os.path.join(root, filename))
                 shutil.copy(os.path.join(root, filename), install_path)
 
-    if os.path.isfile(os.path.join(kbuild_output, "vmlinux")):
-        shutil.copy(os.path.join(kbuild_output, "vmlinux"), install_path)
+    vmlinux_file = os.path.join(kbuild_output, "vmlinux")
+    if os.path.isfile(vmlinux_file):
+        print "found vmlinux, parsing sizes"
+        import elf
+        bmeta.update(elf.read(vmlinux_file))
+        bmeta["vmlinux_file_size"] = os.stat(vmlinux_file).st_size
         bmeta["vmlinux_file"] = "vmlinux"
```

Comment:
Shouldn't we also remove the vmlinux_file from the build.json since it's no longer uploaded?